config/routes.rb

```diff
@@ -66,6 +66,8 @@
     end
     resources :framework_responses, only: %i[update]
 
+    get 'locations_free_spaces', to: 'populations#index'
```

Comment:
Yeah, this feels like our controller class should just be `LocationFreeSpacesController`?

Reply:
There will be more populations related stuff added in the next few stories I'm working on. I'm reluctant to diverge from the adopted front end terminology too.